aztec/src/main/kotlin/org/wordpress/aztec/Aztec.kt

```diff
@@ -43,6 +42,9 @@ open class Aztec private constructor(val visualEditor: AztecText, val toolbar: A
     private constructor(visualEditor: AztecText, sourceEditor: SourceViewEditText,
                 toolbar: AztecToolbar, toolbarClickListener: IAztecToolbarClickListener) : this(visualEditor, toolbar, toolbarClickListener) {
         this.sourceEditor = sourceEditor
+
+        initToolbar()
```

Comment:
shouldn't we remove this line? As it's already executed in the `init{}` block

Reply:
No, that's the fix. The first time `initToolbar()` is callled `sourceEditor` is not initialized yet.

Reply:
oh what a fail from my side! 😂 I thik I'd better leave this for someone else to review